kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/topics/ConnectorsTests.java

```diff
@@ -0,0 +1,96 @@
+package com.provectus.kafka.ui.topics;
+
+import com.provectus.kafka.ui.base.BaseTest;
+import com.provectus.kafka.ui.extensions.FileUtils;
+import com.provectus.kafka.ui.helpers.Helpers;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.*;
+
+public class ConnectorsTests extends BaseTest {
+
+    public static final String LOCAL = "local";
+    public static final String SINK_CONNECTOR = "sink_postgres_activities_e2e_checks";
+    public static final String TOPIC_FOR_CONNECTOR = "topic_for_connector";
+    public static final String TOPIC_FOR_UPDATE_CONNECTOR = "topic_for_update_connector";
+    public static final String TOPIC_FOR_DELETE_CONNECTOR = "topic_for_delete_connector";
+    public static final String FIRST = "first";
+    public static final String CONNECTOR_FOR_DELETE = "sink_postgres_activities_e2e_checks_for_delete";
+    public static final String CONNECTOR_FOR_UPDATE = "sink_postgres_activities_e2e_checks_for_update";
+
+    @BeforeAll
+    @SneakyThrows
+    public static void beforeAll() {
+        Helpers.INSTANCE.apiHelper.createTopic(LOCAL, TOPIC_FOR_CONNECTOR);
+        Helpers.INSTANCE.apiHelper.sendMessage(LOCAL, TOPIC_FOR_CONNECTOR,
+                FileUtils.getResourceAsString("message_content_create_topic.json"), " ");
+    }
+
+    @AfterAll
+    @SneakyThrows
+    public static void afterAll() {
+      Helpers.INSTANCE.apiHelper.deleteTopic(LOCAL, TOPIC_FOR_CONNECTOR);
+    }
+
+    @SneakyThrows
+    @DisplayName("should create a connector")
+    @Test
+    void createConnector() {
+        try {
```

Comment:
will that check ever fail, if it all in try catch?

Reply:
done